<launch>
    <include file="$(find turtlebot3_bringup)/launch/turtlebot3_core.launch" />
    <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" />
    <include file="$(find raspicam_node)/launch/camerav2_410x308_30fps.launch" />
    <node pkg="asl_turtlebot" type="tb3_image_rotate_and_crop.py" name="image_rotater" />
    <node pkg="tf" type="static_transform_publisher" name="camera_true_broadcaster" args="0.03 0 0.067 0.707107 0 0.707107 0 base_footprint raspicam 100" />
    <node pkg="tf" type="static_transform_publisher" name="camera_broadcaster" args="0 0 0 0 0 0.707107 0.707107 raspicam camera 100" />
    <node pkg="tf" type="static_transform_publisher" name="velodyne_broadcaster" args="-0.04 0 0.19 0 0 0 1 base_footprint velodyne 100" />

  
  <node pkg="asl_turtlebot" type="supervisor.py" name="turtlebot_supervisor" output="screen"/>
  <node pkg="asl_turtlebot" type="pose_controller.py" name="turtlebot_pose_controller" />
  <node pkg="asl_turtlebot" type="navigator.py" name="turtlebot_navigator" />
  <node pkg="asl_turtlebot" type="rescue_on_test_sim.py" name="rescue_monitor" />
  <node pkg="asl_turtlebot" type="detector.py" name="turtlebot_detector" />

</launch>